models/contracts/chains/contract_creator_project_mapping/contracts_ethereum_contract_creator_project_mapping.sql

```diff
@@ -0,0 +1,16 @@
+ {{
+  config(
+        tags = ['dunesql'],
+        alias = alias('contract_ethereum_creator_project_mapping'),
+        materialized ='incremental',
+        file_format ='delta',
+        incremental_strategy='merge',
+        unique_key='contract_address',
```

Comment:
unique key should match the test in schema file:
https://github.com/duneanalytics/spellbook/pull/4137/files#diff-6f98b477aba373efb984e25ca4c6291f35b90e24c38090d94db43c38e6debcacR12-R16

please update one or the other to match